app/views/landing/index.html.erb

```diff
@@ -5,9 +5,10 @@
 <input type='submit' value='Search' class='search-btn' />
 
 <div class='restaurants'>
-  <h2>Restaurants:</h2>
-    <ul>
-      <%= link_to 'ThaiTanic', '#' %><br />
-      <%= link_to 'Fu King Chinese', '#' %>
-    </ul>
+  <h2><%= 'Restaurants:' if @restaurants.any? %></h2>
```

Comment:
Why is this conditioned but not the loop on line 10?

Reply:
The loop on line 10 is not conditioned because if there are no restaurants then nothing shows and thats fine, then if there are restaurants the links to the restaurants do show. So, the h2 above needs to be conditioned because it will still show, even if there aren't any restaurants.